cmd/copy.go

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) 2018 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"context"
+
+	"github.com/gardener/etcd-backup-restore/pkg/miscellaneous"
+	"github.com/gardener/etcd-backup-restore/pkg/snapstore"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+// NewSnapshotCommand create cobra command for snapshot
+func NewCopyCommand(ctx context.Context) *cobra.Command {
+	opts := newCopierOptions()
+	var command = &cobra.Command{
+		Use:   "copy",
+		Short: "copy data between buckets",
+		Long:  `Copy data between buckets`,
+		Run: func(cmd *cobra.Command, args []string) {
+			printVersionInfo()
+			logger := logrus.NewEntry(logrus.New())
+			if err := opts.validate(); err != nil {
+				logger.Fatalf("failed to validate the options: %v", err)
+				return
+			}
+			opts.complete()
+
+			source, err := snapstore.GetSnapstore(opts.snapstoreConfig)
```

Comment:
I would suggest putting the logic from this line below in a separate method, this way it could be reused by the `server` command more easily.